CHANGELOG.md

```diff
@@ -44,6 +44,15 @@ Note that 3.1.0 will be the first API stable release and interfaces in this rele
 
 ## API changes
 
+### I/O
+
+* The default fields for reading and writing alignment files were changed to use the `seqan3::field::cigar` instead of
```

Comment:
When reading this, I'm actually not sure how the behaviour of my program might have changed.
Yes, the fields are different, but is the output different? 
So there is no alignment written anymore if it was written before? (and hence the "You can revert this change by..." line?)

Reply:
`The default fields for reading and writing alignment files were changed. Instead of reading the alignment
  information into the `seqan3::field::alignment` it is read into the `seqan3::field::cigar` (both store the same
  information but in different types).
  Hence, reading and writing in one line is now possible without additional reference information:
  `seqan3::alignment_file_output{"foo.sam"} = seqan3::alignment_file_input{"bar.sam"};`
  If you were working with alignment files with default fields and accessed the `seqan3::field::alignment`,
  your code will break. If you still want to read the `seqan3::field::alignment` you need to the alignment field
  manually. Read up on selecting custom fields in our
  [alignment I/O tutorial](http://docs.seqan.de/seqan/3-master-user/tutorial_alignment_file.html).`
Better ?